src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comments } from '../../types/Comments';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comment: Comments[] | null;
+};
+
+export const CommentList: React.FC<Props> = ({ comment: comments }) => {
```

Comment:
is your component `CommetList` takes only one `comment` or it takes some `commentS`?

Reply:
just forget to change it becouse i must to change variable becouse of tests